### DISCUSSION TITLE
Version 23.09.0

### DISCUSSION THREAD

**Discussion by odudex** [selfcustody/krux#240] [Announcements]
This release contains one year of updates created by Krux contributors, from which @tadeubas' work on SD card usage optimizations should be highlighted. Most of the new features are product of ideas and fruitful discussions with the growing community of Krux users, specially from the [Krux Telegram group](https://t.me/SC_Krux).

## Installing

To install this release, please choose your method from the [installation instructions](https://selfcustody.github.io/krux/getting-started/installing/).

To perform an airgapped upgrade (with a microSD card) from a previous signed release, please follow the directions [here](https://selfcustody.github.io/krux/getting-started/installing/from-pre-built-release/#upgrade-via-microsd-card).

### Battery Indicator
Check battery status of M5stickV or Maix Amigo on top right of the screen.

### New Mnemonic From Camera
Use camera as a source of entropy to quickly create a mnemonic.

### Tiny Seed - Export, Print, Punch, Manually Load or Scan
Import and export a binary representation of your mnemonic, in a format popularized by Tiny Seed metal plates. BIP39 mnemonic words number, ranging from 1 to 2048 are punched in binary format on a rectangular grid.
Krux will automatically convert a mnemonic to Tiny Seed format allowing to print or transcript it. You can also load a tiny seed toggling word bits on screen, or make use of machine vision capabilities of K210 chip to directly scan a Tiny Seed mnemonic backup stored on metal or paper.

### Stackbit - Import and Export
Without needing tools, guides or dictionaries, import and export another metal plate backup format, where each of the four digits of the word's number is a sum of marked (punched) numbers 1,2,4 and 8.

### Enter Mnemonic as Word Numbers - Hex and Octal formats
Also available in some metal plate backup formats, you could load your mnemonic words from its decimal BIP39 word number (1-2048), now you can also load from its hexadecimal(0x1-0x800) or octal(01-04000) word number.

### Encryption and Storage
Conveniently store your mnemonics on device's internal flash memory or removable SD card, protecting them with encryption. It is now possible to export encrypted QR codes too.

### Addresses
Beyond verifying your wallet's receive addresses, you can now also list, export and print receive and change addresses.

### SD Card Hot plugging
SD cards can now be inserted and removed at any time, making it easier to use it for signing transactions, messages and storing encrypted mnemonics.

### Transcript Tools for QR codes
Different visualization modes which makes it easier to transcript QR codes.

### Transaction Details
When signing a transaction, more information is presented, ensuring that the user sees all details before signing.

#### Check SD Card
Check if the SD card is detected and explore its content.

#### Delete Mnemonic
Delete any stored encrypted mnemonic, on device's internal flash memory or SD card.

#### Print Test QR
Quickly print a test QR code to check and optimize your printer setup.

#### Create QR Code
Enter a text input to create, print or transcript a QR code that can be later used as an encryption key or as a passphrase.

### Themes
Choose your color theme according to your preference.

### Thermal Printing and CNC
More mnemonic export formats and tools to create and print generic QR codes to be used as passphrases or encryption keys. You can also export QR codes to gcode files and save them in SD cards, allowing you to machine them GRBL compatible CNCs without the need of computers and CAD tools.

#### Persist
Choose where you want to store your settings, on internal flash memory or SD card.

#### Touchscreen
If your device has touchscreen you can change the touch detection threshold.

#### Encoder
Depending on precision, different encoders may require more debounce time to avoid noise or false movements while better encoders can have debounce reduced to allow faster movements.

### Languages
Dutch translations were added.

### UI Tweaks
Small changes to optimize user experience.

### Under the Hood
Small bugfixes, optimizations and code refactoring, targeting better compatibility with coordinator softwares, faster boot and better RAM management.

## Krux Intaller
@qlrd did a fantastic job and made Krux more accessible to general public releasing [Krux Installer](https://github.com/qlrd/krux-installer). Now Krux can be installed with a GUI application, without command line tools. The GUI provides a pleasant and didactic experience to flash all Krux devices, accompanying the user throughout binaries download, signature verification and flashing.

## Krux on Twitter/X
Krux has now an [official Twitter/X account](https://twitter.com/selfcustodykrux), which will managed by @kkdao. Follow and don't miss Krux updates, also don't forget to tag it when sharing your ideas and experiences with Krux devices.

**Comment by tadeubas** [comment]
You forgot to mention under **More Settings**

#### Encoder
If your device has a rotary encoder you can change the debounce threshold.

**Reply by odudex** [reply]
Fixed, however we must add this setting to [documentation](https://selfcustody.github.io/krux/getting-started/settings/) too.
